modules/src/ics04_channel/exported.rs

```diff
@@ -40,6 +40,18 @@ impl State {
             Self::Closed => "CLOSED",
         }
     }
+
+    // Parses the State out from a i32.
+    pub fn from_i32(s: i32) -> Result<Self, error::Error> {
+        match s {
+            0 => Ok(Self::Uninitialized),
```

Comment:
Just out of curiosity, is the Uninitialized state an implementation requirement ? Just asking because in the ics4 spec the Channel state only has 

`enum ChannelState {
  INIT,
  OPENTRY,
  OPEN,
  CLOSED,
}`

I know the Golang implementation has the uninitialized state too, so maybe the spec should be updated ?

Reply:
I think this is an oversight in the cosmos/ICS repo and should be fixed in ICS4. I pinged Chris about it, will update here with details!

Reply:
Opened an issue in https://github.com/cosmos/ics/issues/452.